### PR TITLE
Add calendar widget [SCI-4373][SCI-4371][SCI-4372]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-4371](https://biosistemika.atlassian.net/browse/SCI-4371)
Jira ticket: [SCI-4372](https://biosistemika.atlassian.net/browse/SCI-4372)
Jira ticket: [SCI-4373](https://biosistemika.atlassian.net/browse/SCI-4373)

### What was done
Add calendar widget

### CODE REVIEW

File: app/views/shared/_my_modules_list_partial.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <div class="header">
+      <span class="project" title="<%= task_group[:project_name] %>"><%= task_group[:project_name] %></span>
+      <span class="slash">/</span>
+      <span class="experiment" title="<%= task_group[:experiment_name] %>"><%= task_group[:experiment_name] %></span>
+    </div>
+    <div class="tasks">
+      <% task_group[:tasks].each do |task| %>
+      <div class="task">
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" class="task-icon"><path d="M6.646 14V7.215H4V5h7.938v2.215H9.292V14h4.144c.26 0 .434-.009.542-.022.013-.107.022-.282.022-.542V2.564c0-.26-.009-.435-.022-.542A4.762 4.762 0 0 0 13.436 2H2.564c-.26 0-.435.009-.542.022A4.762 4.762 0 0 0 2 2.564v10.872c0 .26.009.434.022.542.107.013.282.022.542.022h4.082zM2.564 0h10.872c.892 0 1.215.093 1.54.267.327.174.583.43.757.756.174.326.267.65.267 1.54v10.873c0 .892-.093 1.215-.267 1.54-.174.327-.43.583-.756.757-.326.174-.65.267-1.54.267H2.563c-.892 0-1.215-.093-1.54-.267a1.817 1.817 0 0 1-.757-.756C.093 14.65 0 14.327 0 13.437V2.563c0-.892.093-1.215.267-1.54.174-.327.43-.583.756-.757C1.35.093 1.673 0 2.563 0z" fill="#B3B3B3" fill-rule="evenodd"/></svg>
```
Comment (okriuchykhin) [inline app/views/shared/_my_modules_list_partial.html.erb]: Why is it here?

File: yarn.lock [hunk 2]
```diff
@@ -5255,6 +5263,11 @@ jest-docblock@^21.0.0:
   resolved "https://registry.yarnpkg.com/jest-docblock/-/jest-docblock-21.2.0.tgz#51529c3b30d5fd159da60c27ceedc195faf8d414"
   integrity sha512-5IZ7sY9dBAYSV+YjQ0Ovb540Ku7AO9Z5o2Cg789xj167iQuZ2cG+z0f3Uct6WeYLbU6aQiM2pCs7sZ+4dotydw==
 
+jquery@>=1.9:
```
Comment (okriuchykhin) [inline yarn.lock]: Why is it here?

Comment (aignatov-bio) [reply yarn.lock]: It is required by clndr library

File: app/views/shared/_my_modules_list_partial.html.erb (hunk 1 above)
Comment (aignatov-bio) [reply app/views/shared/_my_modules_list_partial.html.erb]: Only inline svg can be styled by css

File: vendor/assets/javascripts/clndr.min.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/*! clndr.min.js v1.5.1 2019-11-27 */
```
Comment (okriuchykhin) [inline vendor/assets/javascripts/clndr.min.js]: Where is the license?

File: app/views/shared/_my_modules_list_partial.html.erb (hunk 1 above)
Comment (mlorb) [reply app/views/shared/_my_modules_list_partial.html.erb]: can you add it to a separate file?

**Pull request merged**